test/test_mimic_explainer.py

```diff
@@ -392,6 +394,29 @@ def test_explain_model_imbalanced_classes(self, mimic_explainer):
         assert np.isclose(surrogate_predictions, model_predictions).all()
         assert global_explanation.method == LIGHTGBM_METHOD
 
+    def test_explain_raw_feats_regression(self, mimic_explainer):
+        # verify that no errors get thrown when calling get_raw_feat_importances
+        num_features = 19999
+        num_rows = 1000
+        test_size = 0.2
+        X, y = make_regression(n_samples=num_rows, n_features=num_features)
+        x_train, x_test, y_train, y_validation = train_test_split(X, y, test_size=test_size, random_state=42)
+
+        lin = LinearRegression(normalize=True)
+        scaler_transformer = Pipeline(steps=[('scaler', StandardScaler())])
+        transformations = ColumnTransformer(transformers=[('num', scaler_transformer, list(range(num_features)))])
```

Comment:
Do you need a column transformer if you're just going to transform all features?

Reply:
hmm agree it is a contrived example,it can only be list of tuples or ColumnTransformer from the doc:

:param transformations: sklearn.compose.ColumnTransformer or a list of tuples describing the column name and
            transformer. When transformations are provided, explanations are of the features before the transformation.
            The format for a list of transformations is same as the one here:
            https://github.com/scikit-learn-contrib/sklearn-pandas.

will try to change it to tuples

Reply:
I seem to have gotten the tuples to work!  I've updated the PR.